tests/testKron.py

```diff
@@ -561,62 +559,50 @@ def testForced(self):
-                        for r, ct in [(radius, ds9.BLUE), (radius*kfac, ds9.CYAN),]:
+                        for r, ct in [(radius, ds9.BLUE), (radius*kfac, ds9.CYAN), ]:
                             shape.scale(r/shape.getDeterminantRadius())
                             ds9.dot(shape, xc, yc, ctype=ct, frame=2)
                     try:
                         self.assertClose(source.get("ext_photometryKron_KronFlux_flux"),
-                                            forced.get("ext_photometryKron_KronFlux_flux"),
-                                            rtol=1.0e-3
-                                        )
+                                         forced.get("ext_photometryKron_KronFlux_flux"),
+                                         rtol=1.0e-3
+                                         )
                         self.assertClose(source.get("ext_photometryKron_KronFlux_radius"),
-                                            scale*forced.get("ext_photometryKron_KronFlux_radius"),
-                                            rtol=1.0e-3
-                                        )
+                                         scale*forced.get("ext_photometryKron_KronFlux_radius"),
+                                         rtol=1.0e-3
+                                         )
                         self.assertEqual(source.get("ext_photometryKron_KronFlux_flag"),
-                                        forced.get("ext_photometryKron_KronFlux_flag")
-                                        )
+                                         forced.get("ext_photometryKron_KronFlux_flag")
+                                         )
                     except:
                         print ("Failed:", angle, scale, offset,
                                [(source.get(f), forced.get(f)) for f in
                                    ("ext_photometryKron_KronFlux_flux",
-                                        "ext_photometryKron_KronFlux_radius",
+                                    "ext_photometryKron_KronFlux_radius",
                                        "ext_photometryKron_KronFlux_flag"
-                                   )
-                               ])
+                                    )
+                                ])
                         raise
 
 
 #-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
 
-def suite():
-    """Returns a suite containing all the test cases in this module."""
-    tests.init()
+class TestMemory(lsst.utils.tests.MemoryTestCase):
+    pass
 
-    suites = []
-    suites += unittest.makeSuite(KronPhotometryTestCase)
-    suites += unittest.makeSuite(tests.MemoryTestCase)
-    return unittest.TestSuite(suites)
 
-def run(exit = False):
-    """Run the tests"""
-    tests.run(suite(), exit)
+def setup_module(module):
+    lsst.utils.tests.init()
 
 if __name__ == "__main__":
-    from argparse import ArgumentParser
```

Comment:
You might want to keep this code in there for the moment. People might expect the command line execution of the test module to not change.